marrow/mailer/message.py

```diff
@@ -258,7 +258,8 @@ def mime(self):
 		return message
 
 	def attach(self, name, data=None, maintype=None, subtype=None,
-		inline=False, filename=None, encoding=None):
+		inline=False, filename=None, filename_charset='', filename_language='',
+		content_encoding=None):
```

Comment:
That's pretty much the only thing stopping me from accepting the pull immediately.  :)  I'll squash when merging, so don't worry about having multiple individual commits.